pkg/common/errors.go

```diff
@@ -27,3 +27,4 @@ const PreemptionPreconditionsFailed = "Preemption preconditions failed"
 const PreemptionDoesNotGuarantee = "Preemption queue guarantees check failed"
 const PreemptionShortfall = "Preemption helped but short of resources"
 const PreemptionDoesNotHelp = "Preemption does not help"
+const NoVictimForRequiredNode = "RequiredNode preemption failed, no victim found"
```

Comment:
Can we improve the wording to not use internal camelcase names? Maybe "No fit on required node, preemption does not help" or something?